dart/constraint/ContactConstraint.hpp

```diff
@@ -132,19 +138,6 @@ class ContactConstraint : public ConstraintBase
   // Documentation inherited
   bool isActive() const override;
 
-  static double computeFrictionCoefficient(
-      const dynamics::ShapeNode* shapeNode);
-  static double computeSecondaryFrictionCoefficient(
-      const dynamics::ShapeNode* shapeNode);
-  static double computeSlipCompliance(
-      const dynamics::ShapeNode* shapeNode);
-  static double computeSecondarySlipCompliance(
-      const dynamics::ShapeNode* shapeNode);
-  static Eigen::Vector3d computeWorldFirstFrictionDir(
-      const dynamics::ShapeNode* shapenode);
-  static double computeRestitutionCoefficient(
-      const dynamics::ShapeNode* shapeNode);
-
```

Comment:
to avoid breaking API, can you keep these functions, even if they aren't used anymore? you could deprecate them instead

Reply:
resolved using global map and mutex